### ISSUE TITLE
4.4.7 - maximizing a window problem

### ISSUE THREAD

**Issue by IvoAndonov** [linuxmint/cinnamon#9104]
[xsession-errors.txt](https://github.com/linuxmint/cinnamon/files/4045589/xsession-errors.txt)

```
 * Cinnamon version (cinnamon --version) 4.4.7
  Daily builds - IDK, whatever is in Arch repos
 * Distribution - Arch
 * Graphics hardware *and* driver used - GTX 1050 Ti 4GB
 * 64-bit
 * Attach /home/<username>/.xsession-errors, or /var/log/syslog
 ```

**Issue**
A minimized window can't be maximized

**Steps to reproduce**
1. Open any window, for instance - the "Change desktop backgroud" window.
2. Click its icon on panel to minimize it.
3. Click it again to maximize - it won't work.

**Expected behaviour**
Clicking a window icon on panel should always maximize it.

**Other information**
I made a video of the problem:
https://www.mediafire.com/file/unwvgg7p1jai78x/simplescreenrecorder-2020-01-10_14.28.53.mkv
This problem WASN'T present in Cinnamon 4.4.6 and it also WASN'T present when I updated the video driver, so I'm pretty sure it's a bug of the DE version.

**Configuration info, if applicable**
If this bug report is related to an Applet, Desklet or Extension, please paste (or use a pastebin service) the offending extension's settings, if it has any.  You can obtain this by opening its configuration, clicking the 'hamburger' button in the upper-right corner of the window, and selecting "Export to a file". **Please be sure to review the contents and remove any personal data it may contain.**

```

Please paste here

```

**Comment by IvoAndonov** [comment]
Adding some more info: found 2 ways to temporarily fix it - DE restart or put the computer to sleep and then wake it up. Computer reboot has the opposite effect - instead of fixing the problem, it brings it back.

**Comment by IvoAndonov** [comment]
Seems to be fixed in Cinnamon 4.4.8. Thank you for fixing it!
